pages/index.vue

```diff
@@ -118,4 +121,17 @@ p {
   display: flex;
   justify-content: center;
 }
+
+@media only screen and (max-width: 768px) {
```

Comment:
Cool, but I think we need some adjustment of font sizes too, see screenshot below @ 660px:
<img width="662" alt="Screen Shot 2020-09-16 at 6 17 37 PM" src="https://user-images.githubusercontent.com/12961270/93408420-558fa600-f849-11ea-8fa1-b56cd60f4429.png">

Also could we use our mixins? E.g.
```suggestion
.floating-card {
  @include lessThan($medium) {
     width: 475px;
  }
}
```